### ISSUE TITLE
Error: Cannot find module 'commander'

### ISSUE THREAD

**Issue by mikwerdna** [JoinColony/colonyStarter#25]
## Description
Step 2 fails to unpack the colony-starter-basic package. Following exception is thrown.

```
internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'commander'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/Users/andy/.config/yarn/global/node_modules/@colony/colony-starter/index.js:4:19)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

## Steps to Reproduce

0. Meet prerequisites listed on [Getting Started](https://github.com/JoinColony/colonyStarter).
1. Execute step 1.
2. As part of step 2, run following command:
```
colony-starter colony-starter-basic
```

**Comment by ryanchristo** [comment]
@mikwerdna Thanks! I was able to reproduce the error after clearing all of the global packages on my machine and reinstalling the `@colony/colony-starter` package.

There were a few more packages we needed to add and they needed to be added to `package.json` in the `packages/colony-starter` directory. We also needed to run `yarn` again from the root to generate an updated `yarn.lock` file.

I created and merged a new pull request (#27) with the necessary fixes. Normally I would have asked you to make the updates yourself but we have a hackathon underway and another just around the corner so I needed to streamline the process a bit. Your contributions are much appreciated!

Try the latest update (run `yarn global upgrade @colony/colony-starter`) and let me know if you are hitting an more errors when running the `colony-starter` script.
